core/cli/src/informant.rs

```diff
@@ -58,8 +58,7 @@ where C: Components {
 		if let Some((ref last_num, ref last_hash)) = last_best {
 			if n.header.parent_hash() != last_hash && n.is_new_best  {
 				let tree_route = ::client::blockchain::tree_route(
-					#[allow(deprecated)]
-					client.backend().blockchain(),
+					|id| client.header(&id)?.ok_or(::client::error::Error::UnknownBlock(format!("Unknown block {:?}", id))),
```

Comment:
```suggestion
					|id| client.header(&id)?.ok_or_else(|| ::client::error::Error::UnknownBlock(format!("Unknown block {:?}", id))),
```

looks like I missed one.